package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "newrelic-koa",
```

Comment:
The package name should use the newrelic scope: `@newrelic/koa` or `@newrelic/koa-instrumentation`.